WordPress/UITestsFoundation/Screens/ActionSheetComponent.swift

```diff
@@ -1,7 +1,6 @@
-import UITestsFoundation
 import XCTest
 
-class ActionSheetComponent: BaseScreen {
+public class ActionSheetComponent: BaseScreen {
```

Comment:
We need to update the screen access level to `public` because it's now part of a different framework than the UI tests that use it. Making it `public` is necessary for the framework consumers to be able to access it.